code/modules/mob/living/simple_animal/hostile/ordeal/white.dm

```diff
@@ -349,10 +350,10 @@
 	maxHealth = 3000
 	health = 3000
 	melee_damage_type = RED_DAMAGE
-	rapid_melee = 2
+	rapid_melee = 6
```

Comment:
...why triple it's melee damage?

Reply:
With all the extra ability spam this gives him, i really don't think he needs a TRIPLED attack speed on top of that. That puts him at 210-270 red damage every 2 seconds. So you're essentially being forced to face tank Goodbye every 4 seconds if you want to melee this guy. That's ludicrous.